app/controllers/admin/time_ranges_controller.rb

```diff
@@ -0,0 +1,55 @@
+module Admin
+  class TimeRangesController < ApplicationController
+    def index
+      @time_ranges = TimeRange.order(updated_at: :desc).page(params[:page])
+    end
+
+    def new
+      @time_range = TimeRange.new
+      render action: :edit
+    end
+
+    def create
+      @time_range = TimeRange.new(time_range_params)
+      if @time_range.save
+        redirect_to admin_time_ranges_path, notice: t('time_range.notice.successfully.created')
+      else
+        flash.now.alert = t('time_range.notice.could_not_be.created')
+        render :edit
```

Comment:
Slight dip in coverage which could be avoided with a test for creating/editing invalid data e.g. with end after start :grinning:

Reply:
Thanks, forgot to check this.